### DISCUSSION TITLE
Modpacks

### DISCUSSION THREAD

**Discussion by JacobRealDev** [cc-tweaked/CC-Tweaked#609] [Q&A] [answered]
# Is there a good modpack for cc?

I like some tech and magics mods that are better with Computer Craft.

**Comment by Lupus590** [comment, marked as answer]
I would suggest looking through the dependents on curseforge: https://www.curseforge.com/minecraft/mc-mods/cc-tweaked/relations/dependents?filter-related-dependents=6

if you are happy to use fabric instead of forge then there's also the restitched fork: https://www.curseforge.com/minecraft/mc-mods/cc-restitched/relations/dependents?filter-related-dependents=6
